Rpackage/R/fitAndScore.R

```diff
@@ -1,6 +1,6 @@
 .fit_and_score <- structure(function(target.mat, feature.mat, 
                                     parameters, n_folds = 3, scorer = NULL, 
-                                    learner = NULL, pruning = TRUE){
+                                    learner = NULL, pruning = TRUE, seed = NULL){
```

Comment:
@parismita 
please remove seed argument; functions typically should not set the random seed; rather the user should do that.

Reply:
actually the sampling is done in fitandscore which is an inline function, so user cannot set its seed....hence I kept it as argument such that if the user wants to set seed it can pass it as argument else it will be random.

If I dont pass it as argument then we cannot make the examples show constant trees as we need to set seed for each sample() function that we use.

Reply:
I don't understand. why can't you just use set.seed in the examples? That is best practice.

Reply:
the random function i used multiple times in fitAndScore function, we need to set the seed multiple times....hence if we do set.seed in example....it will set seed for the topmost sample function...hence ultimately the tree will vary with each run

Also fitAndScore is a helper function.

Reply:
I didnt directly set seed inside the function but kept an option for the user to set seed if they like.

Other thing that can be done I guess is set.seed in example and get.seed inside the function for all sample functions

Reply:
well anyway this is irrelevant to the "tutorals" focus of the PR so no problem